questions/includes/fedora/coding.yml

```diff
@@ -8,8 +8,24 @@ tree:
 
     - title: C
       subtitle: close to the metal
-      href: http://lkml.org
+      href: https://lkml.org
+
+    - title: Haskell
+      subtitle: a nondynamic language for rapid development
+      segue1: Yay, types!
+      segue2: How about...
+      href: https://fedoraproject.org/wiki/Haskell_SIG
```

Comment:
FYI, the ``href`` has no effect if there are ``children`` present.  The ``href`` is only for leaf nodes.